salt/postgresql/server/wal-e.sls

```diff
@@ -120,14 +120,14 @@ wal-e-install:
     - contents_pillar: wal-e:gpg-key
     - user: root
     - group: postgres
-    - mode: 644
+    - mode: "0644"
     - show_diff: False
     - require:
       - pkg: postgresql-server
 
 
 wal-e-gpg-key:
-  cmd.wait:
+  cmd.run:
     - name: gpg --import /var/lib/postgresql/wal-e.gpg
     - user: postgres
     - watch:
```

Comment:
Salts documentation recommends moving from `cmd.wait` with `watch` to `cmd.run` with `onchanges` [here](https://docs.saltproject.io/en/latest/ref/states/all/salt.states.cmd.html#should-i-use-cmd-run-or-cmd-wait). I think we want to change this `watch` directive to be `onchanges`